app/components/WidgetDynamicContent.tsx

```diff
@@ -78,6 +81,10 @@ export default class WidgetDynamicContent extends Component {
         if (requiresFetch) {
             this.fetchData();
         }
+
+        if (widget.definition.postRender && this.containerRef) {
+            widget.definition.postRender(this.containerRef, widget, data.data, this.getToolbox());
```

Comment:
We should probably be doing

```suggestion
            widget.definition.postRender(this.containerRef.current, widget, data.data, this.getToolbox());
```